web/crux-ui/src/components/products/versions/deployments/deployment-events-terminal.tsx

```diff
@@ -8,6 +9,16 @@ interface DeploymentEventsTerminalProps {
 const DeploymentEventsTerminal = (props: DeploymentEventsTerminalProps) => {
   const { events: propsEvents } = props
 
+  const containerRef = createRef<HTMLDivElement>()
+
+  useEffect(() => {
+    if (containerRef.current == null) {
+      return
+    }
+
+    containerRef.current.scrollTop = containerRef.current.scrollHeight
```

Comment:
what happens when the user scroll in to observe some command?

Reply:
It just jumps back to the bottom. I was thinking about a button which could enable/disable auto-scroll and any user input would disable it by default. Ideas?

Reply:
the one we discussed in person would work